internal/test/e2e/plugins.go

```diff
@@ -423,6 +426,12 @@ var VersionCompatibilityOSSPluginConfigurationTests = []VersionCompatibilityPlug
 					Value: "preserve",
 				},
 			},
+			">= 3.1.0": {
```

Comment:
Technically, this is not an update: the version compatibility rule is to remove this `http_response_header_for_traceid` field fold older DPs.

Adding things like this here to verify newer fields are present for newer DPs is not something we have been doing (for example, note that this is not checking that `read_timeout` or `send_timeout` are present for `>=3.0.0`.

I think we should remove this, and maybe add a new test making sure new fields are present for 3.1.0.

Thoughts?

Reply:
+1, good point. Let's test this only when it's a `3.1.0` DP.  That way, we will also avoid modifying it if anything change in 3.2.0+. @omegabytes Note that we also want a test to check if that field is removed for <3.1 DPs.

Reply:
ah nvm, I see it here: https://github.com/Kong/koko/pull/496/files#diff-de20d2d66636a4ebbfeaf31f3cd4f7ef469b1fc14c7d2ab47a0e69efb4ba5191R420 :+1:

Reply:
[1869fc3](https://github.com/Kong/koko/pull/496/commits/1869fc3c2172cc7311650829f2b11c6512886655) asserts a valid config for 310 or newer. Includes some new fields as of 300 as well.